src/silo/storage/dictionary.cpp

```diff
@@ -131,52 +139,62 @@ Dictionary Dictionary::loadDictionary(std::istream& dictionary_file) {
    uint32_t dict_count;
 
    if (!getline(dictionary_file, read_string, '\t')) {
-      std::cerr << "Invalid dict-header1a." << std::endl;
-      return dictionary;
+      throw silo::persistence::LoadDatabaseException(
```

Comment:
Are they really called "header1a"? If yes we should change this.

Reply:
I don't know. I would leave it as it is for now, since we don't know what this does. But yes, at some point, we should probably rename it.